docbook/src/main/doc/adoc/content/chapter-3-JAX_WS_User_Guide.adoc

```diff
@@ -523,23 +523,26 @@ public void testConfigureTimeout() throws Exception
 
 ==== HTTP/2 Support
 
-Starting from JBossWS 7.1.0, the new default CXF HTTP client transport supports HTTP/2 by default.
-The client automatically uses HTTP/2 if the HTTP/2 is supported by server. Otherwise, it falls back to
-HTTP/1.1.
-The client http version can be defined with the `org.apache.cxf.transport.http.forceVersion` property.
-This property can be set to system or bus property. Below is an example illustrating how
-to enforce http version to 1.1:
+Starting from JBossWS 7.1.0,the CXF HTTP client transport has been upgraded to better support HTTP/2.
+The new client which is using java.net.http.HttpClient now seamlessly upgrades to HTTP/2 if supported by the server.
+Otherwise, it falls back to HTTP/1.1. To enable this new client, you must ensure that the
+contextual property `force.urlconnection.http.conduit` is explicitly set to false, either in the System
+properties or the Bus. Otherwise, the default(old) client which based on java.net.HttpURLConnection/javax.net.ssl.HttpsURLConnection
+is enabled.
+
+To http version for the new client can be defined with the `org.apache.cxf.transport.http.forceVersion` property.
+This property can be set to system or bus property. Below is an example illustrating how to enable this new client
+and enforce the http version to 1.1:
 ```
+ BusFactory.getDefaultBus().setProperty("force.urlconnection.http.conduit", false);
  BusFactory.getDefaultBus().setProperty("org.apache.cxf.transport.http.forceVersion", "1");
```

Comment:
so "1" or "1.1."? https://github.com/jbossws/jbossws-cxf/blob/main/modules/testsuite/cxf-tests/src/test/java/org/jboss/test/ws/jaxws/cxf/http2/Http2EndpointTestCaseForked.java#L86

Reply:
Thanks for review. It should be "1.1". Already fixed.